origin-discovery/src/apollo/schema.js

```diff
@@ -65,6 +70,7 @@ const typeDefs = gql`
     subCategory: String!
     price: Price!
     offers: OfferConnection
+    displayType: DisplayType!
```

Comment:
Naming is hard. How about keeping things shorter and just using "display" and "Display" for the type ? It's an enum anyway so when browsing the schema via graphql it is pretty explicit what this is...
`display: Display!`